App/src/components/Result.jsx

```diff
@@ -43,7 +57,13 @@ function Result() {
             </div>
             <Divider />
             <h1 className="text-header text-dPurple">Feedback</h1>
-            {/*The feedback text goes here*/}
+            
+            {/* Feedback Section */}
+            <div className="feedback-section my-4">
+                <h2 className="text-xl font-semibold mb-3">Feedback</h2>
+                <p>{feedback}</p>
```

Comment:
Add the class `"text-dPurple"` to the `<p>` tag to match the style of the rest of the page.